### DISCUSSION TITLE
Process CSS code in HTML <style> elements with Webpack 5

### DISCUSSION THREAD

**Discussion by BlueNebulaDev** [webpack/webpack#15087] [Q&A]
I have an HTML file with style elements containing sass-css code:

```html
<style>
  $font-stack: Helvetica, sans-serif;
  $primary-color: #333;
  
  body {
    font: 100% $font-stack;
    color: $primary-color;
  }
</style>
```

Is it possible to process CSS code in `<style>` elements with Webpack 5?

I'd like to compile them to CSS using SASS and minify the CSS code.

**Comment by BlueNebulaDev** [comment]
Is this use case completely completely unsupported?

I'm playing around with custom elements (web components). I defined many, simple ones. For each custom element I defined a `<template>` element with a simple `<style>` inside. Having to move the styles to a CSS file for each custom element is so uncomfortable. Most `<style>` elements are just a couple of lines of CSS...

Is my approach wrong? How else should I handle this?

**Reply by alexander-akait** [reply]
You need write loader for this case
